index.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Main</title>
+    <!-- LINK GOOGLE FONTS  -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;1,500;1,700&family=Poppins:ital,wght@0,500;0,600;0,700;1,200;1,400;1,500;1,700&family=Roboto:ital,wght@0,100;0,300;0,400;1,100;1,300;1,400&family=Space+Mono&display=swap" rel="stylesheet">
+    <!-- LINK BOOTSTRAP CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-KK94CHFLLe+nY2dmCWGMq91rCGa5gtU4mk92HdvYe+M/SXH301p5ILy+dN9+nJOZ" crossorigin="anonymous">
+    <!-- LINK CSS FILES  -->
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./mobile.css">
+    <link rel="stylesheet" href="./desktop.css">
+</head>
+<body>
+    <section id="hero" class="container-fluid">
+        <header>
+            <div class="contact-icon-container container-fluid p-4 d-md-flex justify-content-end align-items-center d-none">
+                <div class="icons d-flex justify-content-around w-25 align-items-center">
+                    <a href="#/"><i class="fa fa-facebook "></i></a>
+                    <a href="#/"><i class="fa fa-twitter "></i></a>
+                    <span class="">English</span>
+                    <span class="">My Page</span>
+                    <span class="">Log Out</span>
+                </div>
+            </div>
+            <nav class="d-none d-md-block d-md-flex justify-content-between desktop-navbar container-fluid">
+                
+                <div class="header-logo-container">
+                    <a href="./index.html"><img src="./images/tech-conference-logo.webp" class="header-logo" alt=""></a>
+                </div>
```

Comment:
- [ ] Please do not leave blank alt tags in all your images tags. alt tags contain information that is displayed in case your image does not load. This increases your user audience to include people with disabilities such as the blind 😄